packages/api/__tests__/cache.test.js

```diff
@@ -2,9 +2,8 @@ const nock = require('nock');
 const path = require('path');
 const { vol } = require('memfs');
 
-const realFs = jest.requireActual('fs').promises;
+const realFs = jest.requireActual('fs/promises');
```

Comment:
I dropped support for Node 12 in #382 so we can access the promises library this way now.